client/payment-details/summary/index.tsx

```diff
@@ -243,307 +243,314 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 	return (
 		<Card>
 			<CardBody>
-				<div className="payment-details-summary">
-					<div className="payment-details-summary__section">
-						<p className="payment-details-summary__amount">
-							<Loadable
-								isLoading={ isLoading }
-								placeholder="Amount placeholder"
-							>
-								{ formattedAmount }
-								<span className="payment-details-summary__amount-currency">
-									{ charge.currency || 'USD' }
-								</span>
-								{ charge.dispute ? (
-									<DisputeStatusChip
-										status={ charge.dispute.status }
-										dueBy={
-											charge.dispute.evidence_details
-												?.due_by
-										}
-										prefixDisputeType={ true }
-									/>
-								) : (
-									<PaymentStatusChip
-										status={ getChargeStatus(
-											charge,
-											paymentIntent
-										) }
-									/>
-								) }
-							</Loadable>
-						</p>
-						<div className="payment-details-summary__breakdown">
-							{ renderStorePrice ? (
-								<p>
-									{ formatExplicitCurrency(
-										balance.amount,
-										balance.currency
-									) }
-								</p>
-							) : null }
-							{ balance.refunded ? (
-								<p>
-									{ `${
-										disputeFee
-											? __(
-													'Deducted',
-													'woocommerce-payments'
-											  )
-											: __(
-													'Refunded',
-													'woocommerce-payments'
-											  )
-									}: ` }
-									{ formatExplicitCurrency(
-										-balance.refunded,
-										balance.currency
-									) }
-								</p>
-							) : (
-								''
-							) }
-							<p>
+				<Flex direction="row" align="start">
```

Comment:
Although this looks like a large diff, the only change is wrapping the contents of this card in a `<Flex />` component.